### PR TITLE
test: remove old relay test tools part I

### PR DESCRIPTION
**Pull request by araujobarret**
The type of this PR is: **Enhancement**

This PR enhances the morale

### Description

Removes the old createTestEnv which works but is a high-level abstraction not needed thanks to the Relay testing tools we have. This is also one attempt to unblock the Relay 13 upgrade which has failing tests in special with `createTestEnv` and they're very hard to debug and find the issue.

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/v2/DevTools/RootTestPage.tsx [hunk 1]
```diff
@@ -13,6 +13,13 @@ export class RootTestPage {
   // these three properties get hydrated by createTestEnv
   readonly root: ReactWrapper
 
+  constructor(wrapper?: ReactWrapper) {
```
Comment (araujobarret) [inline src/v2/DevTools/RootTestPage.tsx:16]: I kept the PageObject approach but enabled the initialization with a wrapper

File: src/v2/Apps/Order/Routes/__tests__/Accept.jest.tsx [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
-import { graphql } from "react-relay"
+import { graphql, commitMutation as _commitMutation } from "react-relay"
 import {
   acceptOfferFailed,
   acceptOfferInsufficientInventoryFailure,
   acceptOfferPaymentFailed,
   acceptOfferPaymentFailedInsufficientFunds,
   acceptOfferPaymentRequiresAction,
   acceptOfferSuccess,
-  fixFailedPaymentSuccess,
```
Comment (araujobarret) [inline src/v2/Apps/Order/Routes/__tests__/Accept.jest.tsx:18]: this fixture is not needed in this test

**Comment by damassi** [comment]
@araujobarret - is this still WIP or ready to go?

**Comment by araujobarret** [comment]
Ready to go @damassi

**Comment by araujobarret** [comment]
Gonna open more PRs to remove one by one, to not create a massive one directly

**Comment by araujobarret** [comment]
@damassi friendly reminder 🎗️

**Comment by damassi** [comment]
Ah! I'm sorry for dropping the ball on this @araujobarret. Mergin' 👍

**Pull request merged**